integration-tests/src/test/resources/features/AddressAPIHappyPath.feature

```diff
@@ -26,9 +27,9 @@ Feature: Address API happy path test
     #Credential Issue
     When user sends a POST request to Credential Issue end point with a valid access token
     And a valid JWT is returned in the response
-    And JWT lives for two hours
-
-    #Get_Addresses
```

Comment:
If you're removing these line from this feature, then we should have a separate feature test for the `/addresses` endpoint.

Reply:
@merlinc we will create another ticket/feature/PR for this

Reply:
https://govukverify.atlassian.net/browse/OJ-1110

Reply:
Done.